package.json

```diff
@@ -10,6 +10,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "node-fetch": "^3.1.0"
+    "node-fetch": "^3.3.1"
```

Comment:
`node-fetch` can be removed since Node 18 has native `fetch`

Reply:
Feel free to update